docs/infrastructure/operations.md

```diff
@@ -0,0 +1,30 @@
+# Infrastructure Operations
+
+This page summarises the service level objectives of the Infrastructure Team.
+
+In general the Infrastructure Team expect to be able to respond to any issues
+within a couple of days and to have acted on any issues or requests within a
+week. Where possible the team aims to respond much faster than this, however the
+time frame outlined here reflects the volunteer nature of the team.
+
+See [Requesting Infrastructure](./requesting-infrastructure.md) for an overview
+of how to request infrastructure from the Infrastructure Team.
+
+## On Call availability
+
+It is expected that during some events within the competition calendar the
+response time described above will be insufficient. To accommodate this, Teams
+may request that the Infrastructure Team arrange for "on call" availability for
+their event.
+
+On call times can cover the duration of the event, plus a few hours either side,
+with the exact details being agreed ahead of time. (For example at the
+Competition, this would be likely to cover the time in the venue plus some
+buffer but not overnight).
+
+On call availability may be requested up to a month ahead of any event, though
+the Infrastructure Team does not guarantee that it will always be able to
+support every event.
+
+To request on call availability for an event, please follow the same process as
+for requesting any other infrastructure.
```

Comment:
Given there's a dedicated section for it on the "requesting infrastructure" page, it's clearly not exactly the same as "any other infrastructure".

Reply:
It's the same process in that you use what's on that page. This statement is here so that the person knows to go and look at that page for details.

Reply:
a451e0b35aa37df0f943d4e75f4b823886352063 hopefully helps here.